src/Services/DataService.test.ts

```diff
@@ -676,35 +725,80 @@ describe("DataService", () => {
   });
 
   it("has an epic for performing a PUT request", () => {
-    assert.isFunction(fakeService.updateRecordEpic);
+    expect(typeof fakeService.updateRecordEpic).to.equal("function");
   });
 
   describe("updateRecordEpic", () => {
     it("should call updateItem with id and result", () => {
       const onSuccess = spy();
       const expectedResult = { id: "123", fullText: "puppy" };
+      const expectedJSONResult = JSON.stringify(expectedResult.fullText);
       const updateRecordAction = fakeService.actions.updateRecord(expectedResult, { onSuccess });
-      const pushRecordAction = stub(fakeService.actions, "pushRecord");
-      mockAdapter.updateItem.returns(Observable.of(expectedResult));
+      
+      stub(fakeService.serializer, "serialize").returns(expectedJSONResult);
+      
+      fakeService.updateRecordEpic(ActionsObservable.of(updateRecordAction), store)
+        .subscribe(noop, noop,
+          () => {
+            expect(mockAdapter.updateItem.calledWithMatch(expectedResult.id, expectedJSONResult)).to.be.true;
+          });
+    });
+
+    it("updateRecordEpic should call normalize after deserialize", () => {
+      const onSuccess = spy();
+      const expectedResult = { id: "123", fullText: "zella puppy" };
+      const updateRecordAction = fakeService.actions.updateRecord(expectedResult, { onSuccess });
+      const normalizedStub = stub(fakeService.mapper, "normalize");
+
+      stub(fakeService.serializer, "deserialize").returns(expectedResult);
 
       fakeService.updateRecordEpic(ActionsObservable.of(updateRecordAction), store)
         .subscribe(noop, noop,
           () => {
-            assert.isTrue(mockAdapter.updateItem.calledWithMatch(expectedResult.id, JSON.stringify({ fullText: "puppy" })));
+            expect(normalizedStub.firstCall.args[0]).to.equal(expectedResult);
+          });
+    });
+
+    it("updateRecordEpic should call serialize with the results from transform", () => {
+      const onSuccess = spy();
```

Comment:
This and the test below it are testing the exact same thing.  One of them should be testing that `transform` is called.